src/test/ui/break-diverging-value.rs

```diff
@@ -22,8 +22,8 @@ fn get_void() -> Void {
     panic!()
 }
 
-fn loop_break_void() -> i32 { //~ ERROR mismatched types
-    let loop_value = loop { break get_void() };
+fn loop_break_void() -> i32 {
+    let loop_value = loop { break get_void() }; // ok
 }
```

Comment:
cc @rust-lang/types a small bugfix makes some code pass that didn't pass before, because it didn't realize that `get_void()`'s result means the code is unreachable.